src/utils/graphqlClientTemp.js

```diff
@@ -0,0 +1,10 @@
+import ApolloClient from 'apollo-boost';
+import env from './env';
+
+const BENTO_BACKEND = env.REACT_APP_BENTO_BACKEND_API;
+
+const clientBento = new ApolloClient({
+  uri: BENTO_BACKEND,
```

Comment:
Do the canine backend doesn't work?

Reply:
While using canine backing states bar continuously querying to backend for data as it's not getting data from canine Backend for it's query. To suppress this issue for till states bar is refactored we are fetching data from bento backed only for states bar, where studies pages is getting data from canine.

Reply:
The solution here overkills the problem.
You can update AllStatsController to return the empty string and skip the eslint. 
Adding the tmp file will mess up the file structure.  You have to remember to delete this tmp file in the future.  It would be better not to do this.

Reply:
Yeah, I agree with yizhen,  as DevOps need to add a new env variable just for this.

Reply:
Okay, I will make those changes.